### ISSUE TITLE
missing value in sequence

### ISSUE THREAD

**Issue by SheahLinLee** [agartland/seqlogo#3]
Hi,

I have a set of peptide sequence but some of them have missing value such as "AMRE*NSGYSTLT". Is there a way to deal with this to include them in the motif?

Thank you.

**Comment by agartland** [comment]
Yes. When computing the motif you will need to ensure that `'*'` is part of your `alphabet` (see [compute_motif](https://github.com/agartland/palmotif/blob/e534333873defa6d0ed12dd77ed6b4eb2e92474e/palmotif/compute_logo.py#L42)).

This can be a little trickier if you are trying to compute a motif that requires alignment since then the new character also must be given substitution values in the `matrix` (see [compute_pal_motif](https://github.com/agartland/palmotif/blob/e534333873defa6d0ed12dd77ed6b4eb2e92474e/palmotif/compute_logo.py#L237)).

You could also consider replacing the `*` with a gap or some other AA. You may also consider not including these sequences in your motif if they have a stop codon or something else that makes them biologically implausible. Hope that helps!
